nimare/tests/test_stats.py

```diff
@@ -25,3 +25,7 @@ def test_null_to_p():
     assert math.isclose(stats.null_to_p(101, data, "lower"), 0.99)
     assert math.isclose(stats.null_to_p(101, data, "upper"), 0.01)
     assert math.isclose(stats.null_to_p(101, data, "two"), 0.01)
+
+    # modify data to handle edge case
+    data[98] = 100
+    assert math.isclose(stats.null_to_p(100, data, "upper"), 0.01)
```

Comment:
Could you duplicate all (or at least most) of the above tests with the modified array?

Reply:
duplicated relevant edge cases.